README.md

```diff
@@ -124,65 +124,71 @@ practice to alternate morning/evening meetings once SIG has representatives
+| Semantic Conventions: General | Monday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/10xG7DNKWRhxNmFGt3yYd3980a9uwS8lMl2LvQL3VNK8) | [#otel-semantic-conventions-wg](https://cloud-native.slack.com/archives/C041APFBYQP) | [calendar-semconv](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv) | [Armin Ruech](https://github.com/arminru),<br/>[Josh Suereth](https://github.com/jsuereth),<br/>[Reiley Yang](https://github.com/reyang) | [Trask Stalnaker](https://github.com/trask) | 
+| Semantic Conventions: Tooling | Wednesday at 07:00 PT | [Google Doc](https://docs.google.com/document/d/1ygwXgOFRF01UfUOgMr_ElyL7fSDPchTKSrqfj-_HeyA) | [#otel-semantic-conventions-wg](https://cloud-native.slack.com/archives/C041APFBYQP) | [calendar-semconv](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv) | [Josh Suereth](https://github.com/jsuereth) |  | 
+| Semantic Conventions: Messaging | Thursday at 8:00 PT | [Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE) | [#otel-messaging](https://cloud-native.slack.com/archives/C02Q4AAHDSA) | [calendar-semconv](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv) | [Johannes Tax](https://github.com/pyohannes),<br/>[Liudmila Molkova](https://github.com/lmolkova) | [Trask Stalnaker](https://github.com/trask) | 
+| Semantic Conventions: Database | Wednesday at 9:30 PT and Friday at 8:30 PT | [Google Doc](https://docs.google.com/document/d/1zTi_Z4WBisytPnXTpwtnGOK-4Gv5oSTuMHvIQ8--5hg) | [#otel-db-client-semconv-stability-wg](https://cloud-native.slack.com/archives/C06HH05KNMC) | [calendar-semconv](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv) | [Jack Berg](https://github.com/jack-berg),<br/>[Liudmila Molkova](https://github.com/lmolkova) | [Trask Stalnaker](https://github.com/trask) | 
+| Semantic Conventions: System Metrics | Thursday at 07:30 PT | [Google Doc](https://docs.google.com/document/d/1p5TH57t43XpxA48onLzX4PIr3g6ydYKCtR_AUlsCnQk) | [#otel-system-metrics-wg](https://cloud-native.slack.com/archives/C05CTFE9U4A) | [calendar-semconv](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv) | [Josh Suereth](https://github.com/jsuereth) | [Severin Neumann](https://github.com/svrnm) | 
+| Semantic Conventions: LLM | Wednesday alternating between 10:00 PT and 14:00 UTC+8 | [Google Doc](https://docs.google.com/document/d/1EKIeDgBGXQPGehUigIRLwAUpRGa7-1kXB736EaYuJ2M) | [#otel-llm-semconv-wg](https://cloud-native.slack.com/archives/C06KR7ARS3X) | [calendar-semconv](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv) | [Alolita Sharma](https://github.com/alolita),<br/>[Liudmila Molkova](https://github.com/lmolkova) | [Alolita Sharma](https://github.com/alolita) | 
```

Comment:
```suggestion
| Semantic Conventions: LLM | Weekly alternating between Wednesday 10:00 PT and Thursday 14:00 UTC+8 | [Google Doc](https://docs.google.com/document/d/1EKIeDgBGXQPGehUigIRLwAUpRGa7-1kXB736EaYuJ2M) | [#otel-llm-semconv-wg](https://cloud-native.slack.com/archives/C06KR7ARS3X) | [calendar-semconv](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv) | [Alolita Sharma](https://github.com/alolita),<br/>[Liudmila Molkova](https://github.com/lmolkova) | [Alolita Sharma](https://github.com/alolita) | 
```
@lmolkova @nirga Can you confirm? I think in UTC+8 timezone we are on Thursdays.

Reply:
yep, thank you!

Reply:
Thanks, I need to fix that in the sigs.yml first and then apply it to the README, so will not commit your suggestion, but share the commit here

Reply:
Please take a look at https://github.com/open-telemetry/community/pull/2249/commits/61ee9c87747f7cba36bd2aa85a96f1ebfbabc450